front/app/containers/Admin/projects/project/general/components/ProjectCardImageTooltip.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
We could also reuse one component for this one similar to the one above and just pass in props